drivers/video/ov7670.c

```diff
@@ -204,11 +207,11 @@ static const struct video_format_cap fmts[] = {
  * Note that this table assumes the camera is fed a 6MHz XCLK signal
  */
 static const struct ov7670_reg ov7670_init_regtbl[] = {
-	{OV7670_MVFP, 0x20}, /* MVFP: Mirror/VFlip,Normal image */
+	{OV7670_MVFP, 0x00}, /* MVFP: Mirror/VFlip,Normal image */
 
 	/* configure the output timing */
-	/* PCLK does not toggle during horizontal blank, one PCLK, one pixel */
-	{OV7670_COM10, 0x20}, /* COM10 */
+	/* Free running PCLK, default VSYNC, HSYNC and PCLK */
+	{OV7670_COM10, 0x00}, /* COM10 */
```

Comment:
I am just curious about it: Was this not working without this? Is that related to the flipping/mirroring?

Reply:
https://github.com/zephyrproject-rtos/zephyr/commit/3ce3ed38ba462bce1a7a625211da775a43c61351

FRDM-MCXN947: I have that board with me! If I find some time to test it, that would make things easy.

Reply:
Thank you for commenting further on this.

Good idea about unifying the signals polarity. An issue about it sounds like a good first step.

Reply:
cc @danieldegrasse just for visibility as you did the original driver.